cosmrs/Cargo.toml

```diff
@@ -18,6 +18,8 @@ k256 = { version = "0.9", features = ["ecdsa", "sha256"] }
 prost = "0.7"
 prost-types = "0.7"
 rand_core = { version = "0.6", features = ["std"] }
+serde = { version = "1", features = ["serde_derive"] }
+serde_json = "1"
```

Comment:
These are already transitive hard dependencies, so no point in making them optional